### ISSUE TITLE
Web-GUI: german umlauts not working in channel names

### ISSUE THREAD

**Issue by sumulu** [catalinii/minisatip#398]
German umlauts like ü and ö are shown as question mark in column of the stream.
Tested with Google Chrome and Mozilla Firefox.

![umlauts](https://user-images.githubusercontent.com/26624688/33239845-2ebce87e-d2ab-11e7-83b4-b8c02331d7fd.PNG)

Looks like this names are written with JavaScript, I am not sure if utf8 is possible...

**Comment by Jalle19** [comment]
Is this still an issue?

**Comment by Jalle19** [comment]
I'd give it a stab but I can't seem to find a single service with "odd" characters in the service name

**Comment by bccrew** [comment]
@Jalle19 Here is a short TS sample if needed.

Channel is called **één HD**
[Uploading sample.zip…]()

**Comment by Yuri666** [comment]
as workaround i just set web browser codepage to iso-8859-x

**Comment by Jalle19** [comment]
Good to know that's probably just a browser issue, not something being parsed incorrectly from the SDT. I'll have a look.

**Comment by Yuri666** [comment]
just add extension https://chrome.google.com/webstore/detail/set-character-encoding/bpojelgakakmcfmjfilgdlmhefphglae to chrome and choose codepage

**Comment by lars18th** [comment]
Hi @Yuri666 ,

So, yes it works with this extension and selecting the Western (ISO-8859-15). However, it will be preferable to edit the source HTML page, and set the correct values.

**Comment by Jalle19** [comment]
I tried changing the charset but it didn't work. The issue must be deeper.

**Comment by FlexMcMurphy** [comment]
I'm having this problem as well. I have a channel with this character --> É 
It displays fine in TVheadend and Kodi but in the minisatip webui it comes out like �

I tried installing the google extension mentioned above and setting Western (ISO-8859-15) and UTF-8 but that didn't make any difference.
I also edited the minisatip status.html page... I tried:
`<meta http-equiv="Content-Type" content="text/html; charset=utf-8">`
and
`<meta http-equiv="Content-Type" content="text/html;charset="ISO-8859-1"/>`
But still that character comes out wrong on the webui.

Any fix for this or anything else I can try much appreciated!

Flex
